pkg/csi/service/common/util.go

```diff
@@ -146,19 +146,19 @@ func validateVolumeCapabilities(volCaps []*csi.VolumeCapability, validAccessMode
 			}
 		}
 		if !found {
-			return false
+			return fmt.Errorf("Unable to find valid access mode")
```

Comment:
Can we instead say something like - "ReadWriteOnce not supported for file volumes" or "ReadWriteMany not support for block volumes" etc? This is more useful than a generic error message we have here.

Reply:
Done